src/lang/fr.json

```
@@ -76,7 +76,7 @@
   "ch.sbb.handicap": "Personen mit eingeschränkter Mobilität",
   "ch.sbb.infofpw": "Informations changement d'horaire",
   "ch.sbb.infofpw-desc": "L'infographie illustre les défis de chaque gare pour assurer l'exploitation. L'infographie pilote du changement d'horaire de décembre 2018 se limite aux régions de Zurich et de la Suisse orientale.<p>Responsable: P-V-STA-BWP, <a href=\"mailto:xp487@sbb.ch\">xp487@sbb.ch</a></p>",
-  "ch.sbb.infrastruktur": "Infrastructure ferroviaire suisse",
+  "ch.sbb.infrastruktur": "Carte de réseau: infrastructure ferroviaire",
```

```suggestion
  "ch.sbb.infrastruktur": "Carte du réseau des infrastructures ferroviaires",
```
@edellucien  can you confirm ?

---

Yes it is better

---

Lucien said that there is only a minimal difference in meaning between "de" und "du". So, I will leave it the way the translation service provided it.